src/docs/product/integrations/aws-lambda/index.mdx

```diff
@@ -0,0 +1,99 @@
+
+   ![](aws_account.png)
+  
+1. Scroll to the bottom of the create stack page, mark the checkbox at the bottom and press the **Create Stack** button. This will create a CloudFormation stack in the region you are currently in. This stack allows Sentry to instrument your Lambda functions automatically.
+
+   ![](cloudformation.png)
+
+1. Wait a minute or two for the stack to be created. When you see the status change to **Create Complete**, copy your AWS account number, select the region you want to use, and finally press **Next** in the installation modal.
+
+   ![](stack_created.png)
+   
+<Alert title="Note" level="info">
+
+You only need to create the CloudFormation stack once per account even when installing the integration on multiple regions. If you are setting up the integration on a second region, verify that the externalId in the form matches the one for the existing CloudFormation stack on the "Parameters" tab.
+
+</Alert>
+
+   
+1. Review the Lambda functions you want instrumented with Sentry. Lambda functions which are selected will get the Sentry layer and necessary environment variables automatically added when you press **Finish Setup**. Note that you can enable/disable Sentry on Lambda functions after installation as well.
+
+   ![](select_lambdas.png)
+
+1. Depending on how many Lambda functions you enable, you might need to wait a few minutes for this step to complete. If any functions failed to instrument, follow the troubleshooting guide below.
+   
+### Troubleshooting
+
+#### Error `SentryRole already exists`
+
+The stack already exists on your account so you can go to the next step of adding your AWS account information.
+
+#### Error `Invalid existing layer`
+
+Your Lambda function has a layer that no longer exists. Please remove the problematic layer and then enable Sentry on the Lambda in the configuration view for the integration.
+
+#### Error `Please validate the CloudFormation stack was created successfully` 
+
+Sentry is unable to access your AWS account through your CloudFormation stack. This means either the stack has not been created or the external ID used doesn't match the one in your installation. You can go to the Parameters tab of the CloudFormation stack to get the value of the external ID and paste it into the form field.
+
+## Configure
+
+The integration configuration view allows you to do the following with your Lambda functions:
+
+* Enable Sentry
+* Disable Sentry
+* Update the Sentry Lambda Layer to the latest version
+
+In the table view, you will also see the version of the Lambda layer below the name.
+
+   ![](configure.png)
+   
+   
+## Uninstallation
+
+Note that when the integration is uninstalled, your Lambda functions will still be instrumented with Sentry and you will continue to receive errors. If you want to disable Sentry for all your Lambda functions, you should disable Sentry for each function before uninstalling.
+
+
+## How it Works
```

Comment:
This section should moved to a separate page. In the current structure, it's explaining something that's already done (so not useful at this juncture) and is reference content (whereas this page is task-focused content)

Reply:
@PeloWriter what page should I put this then? I feel like this is useful for people who want to know more information about implementation but I don't' want to make the installation docs even longer.

Reply:
I don't see another integration that includes this, so there isn't a current model, but it's good content. Just make it a subpage.